### ISSUE TITLE
Render links for publish and preview once the request is successful

### ISSUE THREAD

**Issue by algomaster99** [IMGIITRoorkee/omniport-frontend-faculty-profile#18]
Currently, we render the hyperlinks to the publish/preview pages in a toast that never closes. This is done because we want to ensure that the faculty doesn't _miss_ the link and they can always click the link on time.

But one drawback to this approach is - let's say the faculty makes multiple edits and after each, they try to preview the page. The UI can easily become cluttered like this:
![Screenshot from 2020-07-16 11-14-48](https://user-images.githubusercontent.com/35191225/87631762-abff3c00-c755-11ea-9fec-808ad202358a.png)

I suggest to just display toast for throwing errors or notifying if it is successful. But where exactly do we render these hyperlinks? I have three approaches in my mind.
1. Pop a tooltip over the button and this tooltip can be faded when the link is clicked.
2. Render "Here is the link" instead of the button and once it is clicked, hide the former and show the latter.
3. We can send them the links via notification.

Open to more suggestions. Let's decide what's the best way to go.

**Comment by Prernaswarna** [comment]
I would like to work on it

**Comment by algomaster99** [comment]
@Prernaswarna yeah, sure. How are you planning to approach this problem? Let's discuss this first.
